Sources/BraintreePayPal/BTPayPalCheckoutRequest.swift

```diff
@@ -58,38 +58,20 @@ import BraintreeCore
 /// Options for the PayPal Checkout flow.
 @objcMembers open class BTPayPalCheckoutRequest: BTPayPalRequest {
 
-    // MARK: - Public Properties
-
-    ///  Used for a one-time payment.
-    ///
-    ///  Amount must be greater than or equal to zero, may optionally contain exactly 2 decimal places separated by '.' and is limited to 7 digits before the decimal point.
-    public var amount: String
-
-    /// Optional: Payment intent. Defaults to `.authorize`. Only applies to PayPal Checkout.
-    public var intent: BTPayPalRequestIntent
-
-    /// Optional: Changes the call-to-action in the PayPal Checkout flow. Defaults to `.none`.
-    public var userAction: BTPayPalRequestUserAction
-
-    /// Optional: Offers PayPal Pay Later if the customer qualifies. Defaults to `false`. Only available with PayPal Checkout.
-    public var offerPayLater: Bool
-
-    /// Optional: A three-character ISO-4217 ISO currency code to use for the transaction. Defaults to merchant currency code if not set.
-    ///
-    /// - Note: See https://developer.paypal.com/docs/api/reference/currency-codes/ for a list of supported currency codes.
-    public var currencyCode: String?
-
-    /// Optional: If set to `true`, this enables the Checkout with Vault flow, where the customer will be prompted to consent to a billing agreement during checkout. Defaults to `false`.
-    public var requestBillingAgreement: Bool
+    // MARK: - Internal Properties
     
-    /// Optional: User email to initiate a quicker authentication flow in cases where the user has a PayPal Account with the same email.
-    public var userAuthenticationEmail: String?
+    var amount: String
+    var intent: BTPayPalRequestIntent
+    var userAction: BTPayPalRequestUserAction
+    var offerPayLater: Bool
+    var currencyCode: String?
+    var requestBillingAgreement: Bool
 
     // MARK: - Initializer
 
     /// Initializes a PayPal Native Checkout request
     /// - Parameters:
-    ///   - amount: Used for a one-time payment. Amount must be greater than or equal to zero, may optionally contain exactly 2 decimal places separated by '.'
+    ///   - amount: Used for a one-time payment. Amount must be greater than or equal to zero, may optionally contain exactly 2 decimal places separated by '.' and is limited to 7 digits before the decimal point.
     ///   - intent: Optional: Payment intent. Defaults to `.authorize`. Only applies to PayPal Checkout.
     ///   and is limited to 7 digits before the decimal point.
```

Comment:
is this part of the `intent` docstring? do we need this comment here?

Reply:
ahh good catch! resolved in 3a8a542bde556b5aa93572f51a7a1a213e24fe1d